samples/bluetooth/hci_spi/prj.conf

```diff
@@ -0,0 +1,12 @@
+CONFIG_GPIO=y
+CONFIG_SPI=y
+CONFIG_MAIN_STACK_SIZE=512
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_MAX_CONN=20
+CONFIG_BT_TINYCRYPT_ECC=n
+CONFIG_BT_RX_BUF_LEN=255
```

Comment:
BT_BUF_RX_SIZE is defined as CONFIG_BT_HCI_RESERVE + CONFIG_BT_RX_BUF_LEN, and HCI_RESERVE is 0 in this case. That value gets used by hci_raw when defining hci_rx_pool and hci_spi when defining cmd_tx_pool, so unclear how that that value can be larger than defined.

The size restriction we currently have is when calling spi_transceive, which is max 255, which is why this config was set to 255 as well.